### DISCUSSION TITLE
Focus for long Anchor text in Menus

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#1968] [Q&A] [answered]
Hey team, seeing some anchor focus wonkiness in Menus when the text is long; i'm not sure if there's a fix on the Paste side though. Was just wondering if the DSYS team just recommends shortening the anchor text?

![image](https://user-images.githubusercontent.com/6936736/138736574-0885b19b-db69-480e-944f-6c1cb88806dd.png)

**Comment by TheSisb** [comment, marked as answer]
Just pushed a PR with a fix https://github.com/twilio-labs/paste/pull/1969.

**Reply by vankhoawin** [reply]
okay that was too quick 🙏

**Reply by TheSisb** [reply]
The PR merged, but it won't be out on NPM until our next release (shortly after [this PR is merged](https://github.com/twilio-labs/paste/pull/1945)).
